nylas/client/restful_models.py

```diff
@@ -663,6 +663,12 @@ def as_json(self):
             dct["when"] = dct["when"].copy()
             dct["when"].pop("object", None)
 
+        if dct.get("participants") and isinstance(dct.get("participants"), list):
+            # The status of a participant cannot be updated and, if the key is
+            # included, it will return an error from the API
+            for participant in dct.get("participants"):
+                participant.pop("status")
```

Comment:
does all participant instance hold "status" value? else wouldn't this cause an error?

Reply:
Good catch! Pushed a commit to address this